mlir/lib/Dialect/Linalg/Transforms/DataLayoutPropagation.cpp

```diff
@@ -447,8 +469,10 @@ bubbleUpPackOpThroughGenericOp(RewriterBase &rewriter, linalg::PackOp packOp,
                             .getDefiningOp<tensor::EmptyOp>()) {
     dest = packOpDest;
   }
+  // Here pack(unpack) isn't naively foldable because the unpack op can be from
+  // an arbitrary domain so we need to keep both.
```

Comment:
picky nit: remove `Here`. I have context, so I can see why you have "here" in this comment. (IIUC, you want to show the difference between two `packGenericOp` calls) However, it looks a little weird to me, if we read the code from the start of this function. How about removing `Here` from the comment?

```suggestion
  // pack(unpack) isn't naively foldable because the unpack op can be from
  // an arbitrary domain so we need to keep both.
```

Reply:
Sounds good!